base/src/main/java/com/getbouncer/cardscan/base/MachineLearningThread.java

```diff
@@ -168,6 +226,30 @@ public void run() {
         });
     }
 
+    private void runModel() {
+        final RunArguments args = getNextImage();
+
+        Bitmap bm;
+        if (args.mFrameBytes != null) {
+            bm = getBitmap(args.mFrameBytes, args.mWidth, args.mHeight, args.mFormat,
+                    args.mSensorOrientation, args.mRoiCenterYRatio);
+        } else if (args.mBitmap != null) {
+            bm = args.mBitmap;
+        } else {
+            bm = Bitmap.createBitmap(480, 302, Bitmap.Config.ARGB_8888);
```

Comment:
We don't need to do this for object detection or do we?

Reply:
If we can skip it then that's better, but the current example is using exactly this (as far as I can tell)